docs/File-Formats.md

```diff
@@ -868,7 +869,74 @@ AKT1 AKT2 10000|AKT<TAB>0.17071492725<TAB>0.264067254391
 ```
 
 
-## Fusion Data  
+## Structural Variant Data
+
+The cBioPortal can load all kinds of structural variant data but at the moment only a subset of them, fusions, are displayed.
+
+### Meta file
+The structural variant metadata file should contain the following fields:
+
+1. **cancer_study_identifier**: same value as specified in [study meta file](#cancer-study)
+2. **genetic_alteration_type**: STRUCTURAL_VARIANT
+3. **datatype**: SV
+4. **stable_id**: structural_variants
+5. **show_profile_in_analysis_tab**: true.
+6. **profile_name**: A name for the fusion data, e.g., "Structural Variants".
+7. **profile_description**: A description of the structural variant data.
+8. **data_filename**: your datafile (e.g. data_SV.txt)
+9. **gene_panel (Optional)**:  gene panel stable id
+
+An example metadata file would be:
+
+```
+cancer_study_identifier: msk_impact_2017
+genetic_alteration_type: STRUCTURAL_VARIANT
+datatype: SV
+stable_id: structural_variants
+show_profile_in_analysis_tab: true
+profile_name: mskimpact2017 SV Data
+profile_description: Structural Variant Data for mskimpact2017
+data_filename: data_SV.txt
+```
+
+### Data file
+A structural variant data file is a tab-delimited file with one structural variant per row.  For each structural variant (row) in the data file, the following tab-delimited values are required:
```

Comment:
There are missing fields here as compared to [cgds.sql](https://github.com/cBioPortal/cbioportal/blob/8d4c7fb40a996ebcfffbcc172e4e03eb6b7cb671/db-scripts/src/main/resources/cgds.sql#L389), like SITE2_EFFECT_ON_FRAME, ENTREZ_GENE_IDs, DNA/RNA_SUPPORT, EXTERNAL_ANNOTATION.  Is this intentional?

Reply:
I added them in. I was not entirely sure about whether this should really be in the model, but we can leave them for now and just let the docs describe the state as is

Reply:
Yeah agreed.  I think they came out of one or two discussions that involved Niki & Trevor.  It sounded like something we might want to have, but I don't think we were entirely sure...it did make it into the model.